src/charts/brush.js

```diff
@@ -60,55 +60,45 @@ export default function Brush (_container) {
     cache.chartBrush.selectAll(".brush-rect")
       .attr("height", config.chartHeight)
 
-    moveBrush()
+    setBrush()
   }
 
-  function getDataExtent () {
+  function getDataExtentUnderBrush () {
     const selection = d3.event.selection
-    const dataExtent = selection.map((d) => invertScale(scales.xScale, d, config.keyType))
-    return dataExtent
-  }
-
-  function clampBrush (_dataExtent) {
-    if (config.keyType === "time") {
-      return [
-        new Date(clamp(new Date(config.brushRangeMin), _dataExtent)),
-        new Date(clamp(new Date(config.brushRangeMax), _dataExtent))
-      ]
-    } else {
-      return [
-        clamp(config.brushRangeMin, _dataExtent),
-        clamp(config.brushRangeMax, _dataExtent)
-      ]
-    }
+    return selection.map((d) => invertScale(scales.xScale, d, config.keyType))
   }
 
-  function moveBrush () {
```

Comment:
I renamed moveBrush to setBrush, as it's only done on init. It's also where it's using brushRangeMin/brushRangeMax instead of data extent

Reply:
you can remove the `clamp` import as it's no longer being used. That should be an ESLint warning or error as well?